src/vsr/superblock.zig

```diff
@@ -521,10 +522,11 @@ pub const FreeSetReference = struct {
     checksum: u128,
     last_block_address: u64,
     last_block_checksum: u128,
-    size: u32,
+    free_set_size: u32,
+    storage_size: u64,
 
     pub fn empty(reference: *const FreeSetReference) bool {
```

Comment:
Assert `storage_size >= data_file_size_min`.
And in the `else` branch below, we can assert that it is strictly greater.

Reply:
Whoa, this is dead code! I definitely should be using this function though, to centralize assertions!